src/inputs/labeled-inputs.scss

```diff
@@ -319,9 +319,9 @@
     }
   }
 
-  > .iui-input,
-  > .iui-textarea,
-  > .iui-select > .iui-select-button {
+  .iui-input,
+  .iui-input-with-icon,
```

Comment:
I think these `padding-bottom` declarations are breaking layout of LabeledSelect

Reply:
Why do you think that? Example?

Reply:
Notice how the text and icon are not vertically centered
![image](https://user-images.githubusercontent.com/9084735/151559146-81c27fe7-babb-487b-ae07-d9bf1acde43c.png)

Reply:
ok so now i'm not sure about this pseudo-element approach again, as it looks really off-center in small and large sizes
![image](https://user-images.githubusercontent.com/9084735/152040864-2eab5b71-0363-419f-808e-7153d1265663.png)

Reply:
@FlyersPh9 for some advice.
Short update: We removed padding and border to `iui-select-button` and added `::after` to add colored bottom border. All items are flex centered and it appears not centered.